Carbon.Cryptography/Carbon.Cryptography.psm1

```diff
@@ -40,7 +40,7 @@ $moduleBinRoot | Out-Null # To make the PSScriptAnalyzer squiggle go away.
 
 $psModulesDirPath = Join-Path -Path $script:moduleRoot -ChildPath 'Modules' -Resolve
 
-Import-Module -Name (Join-Path -Path $psModulesDirPath -ChildPath 'Carbon.Core') `
+Import-Module -Name (Join-Path -Path $psModulesDirPath -ChildPath 'Carbon.Core\Carbon.Core.psm1') `

```

Comment:
Do you need to add `"FlattenModules": true` to the `prism.json` for this to work?

Reply:
No. `FlattenModules` is `true` by default.